src/main/java/de/numcodex/sq2cql/model/common/TermCode.java

```diff
@@ -6,6 +6,7 @@
 import com.fasterxml.jackson.databind.JsonNode;
 
```

Comment:
@alexanderkiel:
have we discussed or thought about whether we need to consider the "version" here as well?